packages/api/src/logger/index.ts

```diff
@@ -0,0 +1,194 @@
+import pino, {
+  BaseLogger,
+  DestinationStream,
+  LevelWithSilent,
+  LoggerOptions,
+  redactOptions,
+} from 'pino'
+
+/**
+ * Determines if log environment is development or test
+ */
+const isDevelopment =
+  process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test'
+
+/**
+ * Determines if log environment is production by checking if not development
+ */
+const isProduction = !isDevelopment
+
+/**
+ * Determines if logs should be prettified.
+ *
+ * Typically if logging to a transport stream or in production
+ * logs will not be prettified.
+ *
+ * In fact, the pino developers note:
+ * "We recommend against using pino-pretty in production,
+ * and highly recommend installing pino-pretty as a development dependency."
+ * See: https://github.com/pinojs/pino-pretty#programmatic-integration
+ *
+ * One exception to this rule may be Netlify functions logging.
+ * Its function logging output readability can benefeit from pretty-printing.
+ */
+export const isPretty = isDevelopment
+
+/**
+ * Defines the necessary pretty printing dependency
+ */
+export const prettifier = require('pino-pretty')
```

Comment:
I would use an ordinary import for this


```suggestion
export { default as prettifier } from 'pino-pretty'
```

Reply:
I doesn't seem to like that:

`import { default as prettifier } from 'pino-pretty'`


```
Could not find a declaration file for module 'pino-pretty'. '/Users/dthyresson/Dropbox (Personal)/projects/redwoodjs/redwood/node_modules/pino-pretty/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/pino-pretty` if it exists or add a new declaration (.d.ts) file containing `declare module 'pino-pretty';`

8 import { default as prettifier } from 'pino-pretty'

```

It seems to like that way to load the dependency -- and this is really only for Dev and if for some reason you want to pretty print in Production like Netlify logs. Ideally, you'd never require this in non-dev.

Reply:
So what that error message is saying is that it can't find TypeScript typings for pino-pretty. I looked, and unfortunately there doesn't seem to exist any in DefinitelyTyped either :( (it would have been at @types/pino-pretty in that case).

What you can do is literally what it says in the error message
> add a new declaration (.d.ts) file containing `declare module 'pino-pretty';`